.config/git/config

```diff
@@ -34,7 +35,10 @@
   undo = reset HEAD^
 [diff]
   algorithm = patience
-  colorMoved = dimmed-zebra
+  # cannot be dimmed-zebra or delta will be unable to distinguish between old and new moved
```

Comment:
this is because dimmed-zebra outputs both types of moved lines with gray and distinguishes them with +-, but the +- don't show up in delta so they are completely indistinguishable. Zebra colors moved lines with two different colors that are different from the normal git green/red